### PR TITLE
feat: static middleware options

### LINKED ISSUE #80: Allow resolution of .html files when no extension is requested
### Feature Use Case
Client-side rendered apps break when file extensions such as .html are present on the URL. I would like to be able to request a route, `/account/test` for example, and have webpack-plugin-serve resolve to `/account/test.html` (if the corresponding html file is present) without changing the URL on the client (i.e. no redirect). It seems the included koa-static middleware allows for this through an options setting 'extensions', which is an array of file extensions to be resolved by koa. I believe the pattern looks like this:

`{ extensions: [ '.html', '.htm'] }`

(honestly, it's unclear to me if the leading period is needed)

### Feature Proposal
Please allow for requesting routes without file-extensions which successfully resolve to matching filenames containing specified extensions (or even allowing the behavior for .html files by default). Ideally, this would happen without altering the URL on the client.

**Example:**
a `/dist` directory has been specified as a static path to resolve from site-root, all files are contained within that directory. within that directory is another directory named `account`. within the `account` directory is a file named `test.html`.

Browser requests: `/account/test`
webpack-plugin-serve finds file: `/account/test.html`
WPS returns the contents of the .html file at route `/account/test`.

Perhaps the most straightforward way to allow for this would be to provide a place to pass options into koa-static along with filesystem paths to resolve?

Thank you!

### PR DESCRIPTION
**Pull request by shellscape**
This PR contains:

- [ ] bugfix
- [x] feature
- [ ] refactor
- [ ] tests
- [x] documentation
- [x] metadata

### Breaking Changes?

- [ ] yes
- [x] no

If yes, please describe the breakage.

### Please Describe Your Changes

Resolves #80. Adds the ability to pass options for `koa-static` via the `builtins.static` middleware.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ declare module 'webpack-plugin-serve' {
   interface Builtins {
     proxy: (args: HttpProxyMiddlewareConfig) => Proxy;
     compress: (opts: CompressOptions) => void;
-    static: (opts: KoaStaticOptions) => void;
+    statik: (paths: Array<String>, opts?: KoaStaticOptions) => void;
```
Comment (shellscape) [inline index.d.ts]: @PlayMa256 going to revert this change. it's `static` and not `statik`. see: https://github.com/shellscape/webpack-plugin-serve/pull/83/files#diff-d7dfcb350720309421bbb465055aafdfR126

Comment (PlayMa256) [reply index.d.ts]: missread that. My bad, reverted.
